x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/actions_log_table.tsx

```diff
@@ -292,22 +291,19 @@ export const ActionsLogTable = memo<ActionsLogTableProps>(
   }) => {
     const getTestId = useTestIdGenerator(dataTestSubj);
     const { pagination: paginationFromUrlParams } = useUrlPagination();
-    const { withOutputs: withOutputsFromUrl } = useActionHistoryUrlParams();
 
     const [expandedRowMap, setExpandedRowMap] = useState<ExpandedRowMapType>({});
 
-    const actionIdsWithOpenTrays = useMemo((): string[] => {
-      // get the list of action ids from URL params on the history page
-      if (!isFlyout) {
```

Comment:
don't we need it ?

Reply:
Nope, we're using the component query params to get the ids anyway. This URL param bit was redundant and was causing the bug, which is why it works as expected on the flyout but not on the page.